db/structure.sql

```diff
@@ -572,6 +573,7 @@ CREATE TABLE public.external_pull_requests (
     external_project_id bigint NOT NULL,
     created_at timestamp(6) without time zone NOT NULL,
     updated_at timestamp(6) without time zone NOT NULL,
+    number integer,
```

Comment:
Just out of curiosity: why aren't we adding a NOT NULL constraint here?

Reply:
Just so that we didn't need to add a default value for old records (this way, old records will have the `number` attribute in `nil` and not some random number that could be misleading).
If we made a backfill, then we could add the constraint

Reply:
Ahhhh I see. Nice!